themes/nekolight/nekolight.theme.sh

```diff
@@ -0,0 +1,25 @@
+#! bash oh-my-bash.module
+
+_omb_theme_nekolight_version='1.0.0'
+_omb_theme_nekolight_symbol=""
```

Comment:
I think we should clarify in `themes/nekolight/README.md` that the user needs to specify a Powerline Font (i.e., a font patched with the Powerline glyphs) to the user's terminal. This is one of the frequently asked questions.